playwright-visual.config.ts

```diff
@@ -0,0 +1,77 @@
+import type { PlaywrightTestConfig } from '@sand4rt/experimental-ct-web';
+import { devices } from '@playwright/test';
+import os from 'os';
+
+const { getBaseUrl } = require('./apps/pie-docs/test/helpers/configuration-helper');
+const baseURL = getBaseUrl();
+process.env.BASE_URL = baseURL;
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+
+  /* Maximum time one test can run for. */
+  timeout: 30 * 1000,
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 5000
+  },
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: os.cpus().length,
```

Comment:
Doesn't seem to be an issue on CI, however you raise a good point when testing locally. We'll modify this to use half of the available CPU cores to avoid local machines grinding to a halt!

Reply:
Nice, the docs have an example of that actually, doing something like this:
```javascript
workers: process.env.CI ? 2 : undefined,
```
`undefined` locally meaning fall back to the default

Reply:
Fixed!